sample/src/main/kotlin/com/uber/autodispose/recipes/subscriberproxies.kt

```diff
@@ -0,0 +1,54 @@
+package com.uber.autodispose.recipes
+
+import com.uber.autodispose.*
+import io.reactivex.disposables.Disposable
+import io.reactivex.exceptions.OnErrorNotImplementedException
+import io.reactivex.plugins.RxJavaPlugins
+
+private val onNextStub: (Any) -> Unit = {}
+private val onErrorStub: (Throwable) -> Unit = { RxJavaPlugins.onError(OnErrorNotImplementedException(it)) }
```

Comment:
I'd like to propose diverging from how RxKotlin does this. The reason is that RxJava actually does special handling for missing error implementations via [`LambdaConsumerIntrospection`](https://github.com/ReactiveX/RxJava/pull/5590), that supplying this would not catch. I'd say for the cases where only a next/success/complete consumer/action is passed in, let's call through to the unhandled one in rxjava. Will comment inline with specifics

Reply:
Hm, interesting. I added RxKotlin's [current implementation](https://github.com/ReactiveX/RxKotlin/pull/141) before `LambdaConsumerIntrospection` was in place, and it matched RxJava's behavior at the time. I'll look into your PR in detail later, but do you think RxKotlin should also be updated?

Reply:
Yeah I'd say it probably should. Would be a non-breaking change, but it would allow rxjava to properly expose the introspection details